system/lib/libc/emscripten_internal.h

```diff
@@ -123,6 +123,8 @@ void _emscripten_throw_longjmp(void);
 
 void __handle_stack_overflow(void* addr);
 
+void __emscripten_atomics_sleep(int);
```

Comment:
Should this take a `double` since that is that type expected by `Atomics.wait` and passed to `emscripten_thread_sleep`

Reply:
I guess it should yeah.